common/web/types/test/ldml-keyboard/test-ldml-keyboard-xml-reader.ts

```diff
@@ -36,7 +36,13 @@ describe('ldml keyboard xml reader tests', function () {
       callback: (data, source, subpath, callbacks) => {
         assert.ok(source?.keyboard3?.keys);
         const k = pluckKeysFromKeybag(source?.keyboard3?.keys.key, ['a', 'b', 'c']);
-        assert.sameDeepOrderedMembers(k, [
+        assert.sameDeepOrderedMembers(k.map((entry) => {
+          // Drop the Symbol members from the returned keys; assertions may expect their presence.
```

Comment:
The Chai version update caused the corresponding unit tests to fail; the "actual" objects from `pluckKeysFromKeybag` have `Symbol.___` members, while the "expected" ones specified below do not.

In the previous Chai version, `Symbol.___` members were not checked.

Reply:
@srl295